library/iFixit/Matryoshka/McRouter.php

```diff
@@ -5,7 +5,7 @@
 use iFixit\Matryoshka;
 
 class McRouter extends Memcached {
-   const MCROUTER_INVALID_CHARACTERS = " \r\n";
+   private const MCROUTER_INVALID_CHARACTERS_REGEX = "[ \r\n]";
```

Comment:
Hmm.. shouldn't *this* get the same regex as Memcached? The allowed keys are *more* restrictive on McRouter cause we have to use the ascii protocol I think. At the very least, I'd expect this to be the same as the Memcached bad chars regex.

Reply:
Fair enough - I think I kept it the same as I was trying to not be invasive - but yes you are correct it's still Memcached - same rules apply

Reply:
This also isn't a valid regex, so how does this work?